code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -130,7 +137,10 @@
 	new /obj/item/weapon/reagent_containers/syringe(src)
 	new /obj/item/weapon/reagent_containers/glass/bottle/ether(src)
 	new /obj/item/weapon/storage/pill_bottle/psychiatrist(src)
-
+	new /obj/random/plushie(src)
+	for(var/i = 0, i < 3, i++)

```

Comment:
for(var/i in 0 to 3)

Reply:
I'd recommend `1 to 4`, personally, as the standard is starting with `1` as opposed to `0`.